Jenkinsfile

```diff
@@ -46,6 +46,13 @@ git merge ${merge_opts} '${commit}' -m '${commit_msg}'
 """)
 }
 
+// https://issues.jenkins-ci.org/browse/JENKINS-33511
+def set_workspace() {
+    if(env.WORKSPACE.toString() == 'null') {
```

Comment:
If `pwd` works while we are in the workspace of the job then its' probably the best solution.